assets/js/blocks/rating-filter/block.tsx

```diff
@@ -4,6 +4,7 @@
 import { __, sprintf } from '@wordpress/i18n';
 import { speak } from '@wordpress/a11y';
 import Rating from '@woocommerce/base-components/product-rating';
+import { Notice } from '@wordpress/components';
```

Comment:
For components that load on the front end, we should use `wordpress-components` instead of `@wordpress/components`.

Reply:
Updated! Thanks for the heads up and also for sharing more details about the reason why we are keeping both dependencies in place.

The context for anyone else following this convo: The newer version of `@wordpress/components` has a performance impact on WooCommerce Blocks, as it includes libraries that increase the bundle size significantly. For this reason, we are relying on `wordpress-components` for now.